plasma_framework/contracts/src/exits/utils/BondSize.sol

```diff
@@ -68,7 +79,19 @@ library BondSize {
         }
     }
 
-    function validateBondSize(Params memory self, uint128 newBondSize) private view {
+    /**
+    * @notice Returns the current exit bounty size
+    */
+    function exitBountySize(Params memory self) internal view returns (uint128) {
+        if (now < self.effectiveUpdateTime) {
+            return self.previousExitBountySize;
+        } else {
+            return self.updatedExitBountySize;
+        }
+    }
+
+    function validateBondSize(Params memory self, uint128 newBondSize, uint128 newExitBountySize) private view {
+        require(newBondSize >= newExitBountySize, "Incorrect Exit Bounty Size");
```

Comment:
minor: could have better error message telling it needs bond size to be >= bounty

Reply:
done `76e2c7b`